contracts/src/libraries/PlonkVerifier.sol

```diff
@@ -262,31 +261,30 @@ library PlonkVerifier {
             mstore(add(dataPtr, 0x600), mload(add(pi, 0x80))) // PI[4]
             mstore(add(dataPtr, 0x620), mload(add(pi, 0xa0))) // PI[5]
             mstore(add(dataPtr, 0x640), mload(add(pi, 0xc0))) // PI[6]
-            mstore(add(dataPtr, 0x660), mload(add(pi, 0xe0))) // PI[7]
 
             // proof
             let wire0Ptr := mload(proof)
-            mstore(add(dataPtr, 0x680), mload(wire0Ptr)) // wire0.x
-            mstore(add(dataPtr, 0x6a0), mload(add(wire0Ptr, 0x20))) // wire0.y
+            mstore(add(dataPtr, 0x660), mload(wire0Ptr)) // wire0.x
+            mstore(add(dataPtr, 0x680), mload(add(wire0Ptr, 0x20))) // wire0.y
             let wire1Ptr := mload(add(proof, 0x20))
-            mstore(add(dataPtr, 0x6c0), mload(wire1Ptr)) // wire1.x
-            mstore(add(dataPtr, 0x6e0), mload(add(wire1Ptr, 0x20))) // wire1.y
+            mstore(add(dataPtr, 0x6a0), mload(wire1Ptr)) // wire1.x
+            mstore(add(dataPtr, 0x6c0), mload(add(wire1Ptr, 0x20))) // wire1.y
             let wire2Ptr := mload(add(proof, 0x40))
-            mstore(add(dataPtr, 0x700), mload(wire2Ptr)) // wire2.x
-            mstore(add(dataPtr, 0x720), mload(add(wire2Ptr, 0x20))) // wire2.y
+            mstore(add(dataPtr, 0x6e0), mload(wire2Ptr)) // wire2.x
+            mstore(add(dataPtr, 0x700), mload(add(wire2Ptr, 0x20))) // wire2.y
```

Comment:
can you also explain this value change?

Reply:
yes, so we removed a public input field (from 8 to 7), 
so all these values, which were laying out consecutively in the memory space, are moved up in their memory location by `0x20` (one field element is 32 bytes)